addons/website_blog/static/src/js/website_blog.inline.discussion.js

```diff
@@ -50,6 +51,21 @@ var InlineDiscussion = core.Class.extend({
                         'marginLeft': "+=250px"
                     });
                 }
+            } else if ($(event.target).hasClass('discussion-link')) {
```

Comment:
@Gorash If you are asking to concatenate else if condition in to first if stamtent then note that, first we add some marginLeft and in else if we are removing that marginLeft